.travis.yml

```diff
@@ -132,6 +132,18 @@ matrix:
     env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet6
     bundler_args:
     script: bundle exec rake beaker
+  - rvm: 2.4.4
+    sudo: required
+    services: docker
+    env: BEAKER_set="ubuntu-2004" BEAKER_PUPPET_COLLECTION=puppet5
```

Comment:
This fails because there is no http://apt.puppet.com/puppet5-release-focal.deb.

Can we drop test puppet5 on focal ?

Reply:
Hi,

We had a similar issue in https://github.com/ghoneycutt/puppet-module-rpcbind/pull/48#issuecomment-658826023 and there we decided to drop Puppet 5 support/tests for that specific operating system.

Reply:
That's the problem with check for Ubuntu 20.04. You're wright. So I have removed puppet5 in this specific version.

But this is not the only problem, for example, in the `BEAKER_set="debian-8" BEAKER_PUPPET_COLLECTION=puppet6` test is failing with the error:
```
$ bundle exec rake beaker

TEST_TIERS env variable not defined. Defaulting to run all tests.

/home/travis/.rvm/rubies/ruby-2.5.1/bin/ruby -I/home/travis/build/ghoneycutt/puppet-module-pam/vendor/bundle/ruby/2.5.0/gems/rspec-core-3.9.2/lib:/home/travis/build/ghoneycutt/puppet-module-pam/vendor/bundle/ruby/2.5.0/gems/rspec-support-3.9.3/lib /home/travis/build/ghoneycutt/puppet-module-pam/vendor/bundle/ruby/2.5.0/gems/rspec-core-3.9.2/exe/rspec spec/acceptance

An error occurred while loading ./spec/acceptance/00_pam_spec.rb.

Hint: Install the `did_you_mean` gem in order to provide suggestions for similarly named files.

Failure/Error: require 'beaker-rspec'
```
I think this is the error in all tests now.